_source/_posts/2022-10-11-angular-dependency-injection.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: blog_post
+title: "Practical Usages of Dependency Injection in Angular"
```

Comment:
I'm not going to edit the title, it's not wrong, and my comment is opinionated for sure, but I'm not a huge fan of "usage" - use is so much simpler/more direct, and since you're going for the plural, I'd recommend either "Practical Usage of Dependency Injection in Angular" (because usage can imply the plural without adding a syllable) OR "Practical Uses of Dependency Injection in Angular" - the scores are all identical

Reply:
I'm good with "uses" I'll make the change